### DISCUSSION TITLE
Remove "Taiwan" from country_codes_to_names in the analytics/lib/plausible/stats/countries.ex.

### DISCUSSION THREAD

**Discussion by donniek** [plausible/analytics#1138] [Anything Else]
It is common sense that Taiwan is not a country. Taiwan is a part of China.
A excellent team should not be so low-level mistakes.
Please remove "Taiwan" from country_codes_to_names in the analytics/lib/plausible/stats/countries.ex immediately.

**Comment by g105b** [comment]
Ask any Taiwanese person and they'll tell you it is indeed a country. I understand that the political situation is complicated over there, but I don't think it's as black and white as you make out.

**Comment by ukutaht** [comment]
The countries that are listed in Plausible come from the International Organization for Standardization  which [lists Taiwan as a country](https://en.wikipedia.org/wiki/ISO_3166-2:TW).

If there's a better standard than ISO, please recommend it and we will consider using it.

Meanwhile we will not manually change the list of countries to satisfy one side of a political disagreement. I will lock this discussion. Sorry if this upsets anyone.
